### PR TITLE
Added configuration example to do Azure AD EAP-TLS Machine

### PR DESCRIPTION
**Pull request by fdurand**
Authentication

# Description
Azure AD EAP-TLS Machine authentication documentation (for PR  #7889)

# Impacts
No

# Delete branch after merge
YES

# Checklist
(REQUIRED) - [yes, no or n/a]
- [ ] Document the feature
- [ ] Add OpenAPI specification
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries

## Enhancements
* Documentation for Azure AD EAP-TLS machine Authentication

### CODE REVIEW

File: docs/installation/azuread.asciidoc [hunk 1]
```diff
@@ -79,3 +80,48 @@ You can perform 802.1X authentication of users using Azure AD but this will only
 . Restart radiusd using `/usr/local/pf/bin/pfcmd service radiusd restart`
 . All the users matching this realm will now authenticate against Azure AD. Make sure you also have a connection profile with auto-registration enabled and the Azure AD source in it so that your users are correclty authorized when connecting.
 
+===== Using Azure AD EAP-TLS machine authentication
+
+You can perform a EAP-TLS authentication and verify the machine group membership in order to provide a access to the network.
```
Comment (extrafu) [inline docs/installation/azuread.asciidoc:85]: perform a EAP-TLS -> perform EAP-TLS
provide a access to the network -> provide network access

File: docs/installation/azuread.asciidoc [hunk 2]
```diff
@@ -79,3 +80,48 @@ You can perform 802.1X authentication of users using Azure AD but this will only
...
+
+To do that first you will have to provide to the end device a certificate that contains the Device ID, to do this go in the Intune management interface and configure the template like this:
+
+image::Intune_SCEP_machine.png[scaledwidth="100%",alt="Intune SCEP machine configuration"]
+
+As you can see the CN (Common Name) will contain the Device Identifier, so when the device will connect on the secure ssid, the username will be equal to the device ID (like 8df07f7e-d98e-4579-aa97-bfcfaaa7fe38)
```
Comment (extrafu) [inline docs/installation/azuread.asciidoc]: will connect on the secure ssid -> connects on the secure SSID

File: docs/installation/azuread.asciidoc [hunk 3]
```diff
@@ -79,3 +80,48 @@ You can perform 802.1X authentication of users using Azure AD but this will only
...
+
+Now it just a matter to retrieve the groupmembership associated with the device ID, in order to do that you will need to change the "User Groups Url" parameter in the "Azure Active Directory" authentication source.
```
Comment (extrafu) [inline docs/installation/azuread.asciidoc]: groupmembership -> group membership
User Groups Url -> User Groups URL

File: docs/installation/azuread.asciidoc [hunk 4]
```diff
@@ -79,3 +80,48 @@ You can perform 802.1X authentication of users using Azure AD but this will only
...
+
+The URL will be `https://graph.microsoft.com/v1.0/devices(deviceId='%USERNAME')/memberOf` (for more information https://learn.microsoft.com/en-us/graph/api/device-list-memberof?view=graph-rest-1.0&tabs=http)
+
+In this example the api call will be `https://graph.microsoft.com/v1.0//devices(deviceId='8df07f7e-d98e-4579-aa97-bfcfaaa7fe38')/memberOf?$select=id,displayName and the reply is:
```
Comment (extrafu) [inline docs/installation/azuread.asciidoc]: api -> API
reply is -> reply will be

File: docs/installation/azuread.asciidoc [hunk 5]
```diff
@@ -79,3 +80,48 @@ You can perform 802.1X authentication of users using Azure AD but this will only
...
+      {
+         "@odata.type""=>""#microsoft.graph.group",
+         "id""=>""1bced11e-0cae-4689-8afa-060ec0b3341f",
+         "displayName""=>""ZaymLed - 3D software"
+      },
+   ],
+   "@odata.context""=>""https://graph.microsoft.com/v1.0/$metadata#directoryObjects(id,displayName)"
+}`
+
+In the "Azure Active Directory" authentication source create a authentication rule like this:
```
Comment (extrafu) [inline docs/installation/azuread.asciidoc]: "In the "Azure Active Directory" authentication source create a authentication rule like this:
" -> "From the "Azure Active Directory" authentication source, create an authentication rule like this:"

**Comment by extrafu** [comment]
@fdurand Review done, minor changes required and ready to be merged after.

**Comment by fdurand** [comment]
BUMP

**Pull request merged**